Sources/PriorityQueueModule/PriorityQueue.swift

```diff
@@ -214,6 +224,7 @@ public struct PriorityQueue<Element: Comparable> {
 
   // MARK: -
 
+  @inline(__always)
```

Comment:
Should we add this attribute to `_bubbleUp(startingAt:)` as well?

Reply:
I added it, saw no major difference - but it's part of ca63256 too.